webview/src/plots/components/PlotsContainer.tsx

```diff
@@ -173,11 +203,25 @@ export const PlotsContainer: React.FC<PlotsContainerProps> = ({
             </div>
           </Tooltip>
         </summary>
+        {changeNbItemsPerRow && hasItems && (
+          <div
+            className={styles.nbItemsPerRowSlider}
+            data-testid="nb-items-per-row-slider"
+          >
+            <MinMaxSlider
+              minimum={1}
+              maximum={setMaxNbPlotsPerRow}
+              label="Number of plots per row"
```

Comment:
[C] I do like the idea of reversing this, giving the label `Plot Width` and having the min value = max number of plots / max = 1 plot per row.

Reply:
underlying implementation would be the same, just change the way the slider operates.

Reply:
Yep, sounds good to me too.